oci/runtime_oci.go

```diff
@@ -48,10 +51,14 @@ type runtimeOCI struct {
 
 // newRuntimeOCI creates a new runtimeOCI instance
 func newRuntimeOCI(r *Runtime, handler *RuntimeHandler) RuntimeImpl {
+	runRoot := defaultRunRoot
```

Comment:
Is the reason why we do this here and not during config validation because the issue can affect already running containers?

If so, then I'd be fine with that. If not I'd suggest setting the runRoot once on configuration validation or even before on CLI argument parsing.

Reply:
I liked doing it here because it can be the default only for runtime_oci, and I didn't want to effect any other runtimes.

Reply:
I see. I would say it is better to keep it there since with that solution already running workloads still work as expected.

Reply:
sorry--do you mean you'd prefer to move this check into validation? 'there' isn't clear to me

Reply:
No, it's fine like it is now. :)

Reply:
SGTM!